README.md

```diff
@@ -318,6 +319,10 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
 * [Presentation slides for QA applications of MLNs](https://github.com/clulab/nlp-reading-group/blob/master/fall-2015-resources/Markov%20Logic%20Networks%20for%20Natural%20Language%20Question%20Answering.pdf)
 * [Presentation slides](https://github.com/clulab/nlp-reading-group/blob/master/fall-2015-resources/poon-paper.pdf)
 
+### Text Summarization
+* [TextRank- bringing order into text](http://web.eecs.umich.edu/~mihalcea/papers/mihalcea.emnlp04.pdf) by Mihalcea and Tarau is regarded as the first paper on text summarization. The code is available [here](https://github.com/ceteri/pytextrank)
+* [Modelling compressions with Discourse constraints](http://jamesclarke.net/media/papers/clarke-lapata-emnlp07.pdf) by Clarke and Zapata provides a discourse informed model for summarization and subtitle generation.
```

Comment:
Hey @the-ethan-hunt ! 
Thanks for the PR! 

Any particular reason for going all the way back to EMNLP 2007? Unless this is a foundational paper (like Mikolov's 2010 word2vec paper for text embeddings) - I am inclined to include something more effective and recent. 

In an_ideal universe_, I'd like to see the state of the art papers and code on both extractive and abstractive summarization.

Reply:
@NirantK , I took that paper from this [list](https://github.com/mhagiwara/100-nlp-papers). Other papers, as I mentioned don't have open-sourced PDFs or documents. I have seen papers on aforementioned topics but I fear they are not freely available.

Reply:
Hey @the-ethan-hunt , as the maintainer of that list mentions he picked those papers from a really old Quora question. A good starting point when looking for open access papers from the top conferences is [arXiv Sanity](http://www.arxiv-sanity.com/). 

For instance, [arXiv Sanity search on Text Classification](http://www.arxiv-sanity.com/search?q=text+summarization) quickly leads me to [this EMNLP 2017 paper](https://arxiv.org/pdf/1708.00625v1.pdf) which does abstractive summarization using deep regenerative decoders. 

Another strategy is to look for beginner friendly tutorials in popular packages like Keras or Tensorflow. A quick 30-second search led me to [textsum](https://github.com/tensorflow/models/tree/master/research/textsum) on the official Tensorflow repository. 

I hope this helps you broaden your search toolkit a little bit. 

I have mentioned 2 abstractive approaches above. I encourage you to look for extractive approaches as well as work which preferably has code released with it.